src/components/cursor-controller.js

```diff
@@ -38,33 +38,36 @@ AFRAME.registerComponent("cursor-controller", {
     this.origin = new THREE.Vector3();
     this.direction = new THREE.Vector3();
     this.controllerQuaternion = new THREE.Quaternion();
+    this.activeTouch = null;
 
     this.data.cursor.setAttribute("material", { color: this.data.cursorColorUnhovered });
 
-    this._handleMouseDown = this._handleMouseDown.bind(this);
-    this._handleMouseMove = this._handleMouseMove.bind(this);
-    this._handleMouseUp = this._handleMouseUp.bind(this);
-    this._handleWheel = this._handleWheel.bind(this);
-    this._handleEnterVR = this._handleEnterVR.bind(this);
-    this._handleExitVR = this._handleExitVR.bind(this);
-    this._handlePrimaryDown = this._handlePrimaryDown.bind(this);
-    this._handlePrimaryUp = this._handlePrimaryUp.bind(this);
-    this._handleModelLoaded = this._handleModelLoaded.bind(this);
-    this._handleCursorLoaded = this._handleCursorLoaded.bind(this);
-    this._handleControllerConnected = this._handleControllerConnected.bind(this);
-    this._handleControllerDisconnected = this._handleControllerDisconnected.bind(this);
-
-    this._handleTouchStart = this._handleTouchStart.bind(this);
-    this._updateRaycasterIntersections = this._updateRaycasterIntersections.bind(this);
-    this._handleTouchMove = this._handleTouchMove.bind(this);
-    this._handleTouchEnd = this._handleTouchEnd.bind(this);
-
-    this.el.sceneEl.renderer.sortObjects = true;
-    this.data.cursor.addEventListener("loaded", this.cursorLoadedListener);
+    const functionNames = [
+      "_handleTouchStart",
```

Comment:
- https://github.com/mozilla/hubs/pull/372#discussion_r186886185
- https://github.com/mozilla/hubs/pull/372#discussion_r186895932

although it was previously 15 lines, this is now 20 lines. using an array of strings increases the chances for linting mistakes, per @netpro2k's comment. and to my comment, it increases the minified bundle size because the strings won't be renamed to single-character function names.

Reply:
Changed it back to the way it was before